.github/workflows/notebook-test.yml

```diff
@@ -86,12 +86,10 @@ jobs:
         shell: python
         run: |
           import subprocess
-          subprocess.run([
-              "echo",
-              "c.InlineBackend.figure_format = 'svg'",
-              ">>",
-              "~/.ipython/profile_default/ipython_kernel_config.py"
-          ])
+          subprocess.run(
+            "echo \"c.InlineBackend.figure_format = 'svg'\" >> ~/.ipython/profile_default/ipython_kernel_config.py",
+            shell=False
```

Comment:
This would need to be `shell=True`. But better is to use `Path(...).write_text()`

Reply:
Always get it confused 😢 
Not sure it's actually the right file; need to work out which config file it's reading from

Reply:
I discovered 84b6514176e800675ddee54b2e1ccf14714513be which is much cleaner and means writers can produce them locally without changing their global config settings.